spec.go

```diff
@@ -16,12 +16,19 @@ package connector
 
 import (
 	"github.com/conduitio/conduit-connector-google-cloudstorage/config"
+	"github.com/conduitio/conduit-connector-google-cloudstorage/source"
 	sourceConfig "github.com/conduitio/conduit-connector-google-cloudstorage/source/config"
 	sdk "github.com/conduitio/conduit-connector-sdk"
 )
 
+var Connector = sdk.Connector{
+	NewSpecification: specification,
+	NewSource:        source.NewSource,
+	NewDestination:   nil,
+}
+
 // Specification returns the connector's specification.
```

Comment:
Specification, `S` should be lowercase right?

Reply:
:) Done